pkg/daemon/daemon.go

```diff
@@ -66,6 +71,7 @@ type Daemon struct {
 	// fusedev shared mode: zero, one or more RAFS instances
 	// fscache shared mode: zero, one or more RAFS instances
 	Instances rafsSet
+	authCache *lru.Cache
```

Comment:
Auth should be shared among different nydusd daemons. So why to bind the cache to a single nydusd daemon?

Reply:
Fixed. Move it to the manager.